apis/elbv2/v1beta1/ingressclassparams_types.go

```diff
@@ -41,6 +41,22 @@ const (
 	LoadBalancerSchemeInternetFacing LoadBalancerScheme = "internet-facing"
 )
 
+// SubnetSelector selects one or more existing subnets.
+type SubnetSelector struct {
+	// ID specifies the resource ID of a subnet. Exactly one of this or `tagKey` must be specified.
+	// +kubebuilder:validation:Pattern=subnet-[0-9a-f]+
+	// +optional
+	ID string `json:"id,omitempty"`
+
+	// TagKey specifies all subnets containing a tag with the given key and the value specified in `tagValue`.
+	// Exactly one of this or `id` must be specified.
+	// +optional
+	TagKey string `json:"tagKey,omitempty"`
```

Comment:
Since the describe is on a single tag key (on the given vpc), if there are multiple matching subnets per AZ, we make the tag feature unusable. It is more usable if we support multiple tag keys

Reply:
The admin presumably has control over subnet tagging, so they can avoid putting the tag value on more than one subnet in an AZ.

We could make this:

```go
type TagSelector struct {
    Key string
    Value string
}

type SubnetSelector struct {
    ID string
   Tags []TagSelector
```

to be fully generalized, but that seems overkill.